src/settings-web/src/components/GeneralSettings.tsx

```diff
@@ -121,14 +121,18 @@ export class GeneralSettings extends React.Component <any, any> {
         }
         <Separator />
         <Text variant='xLarge'>General</Text>
-        {!this.state.settings.general.packaged &&
-        (
+        <Stack>
+          {this.state.settings.general.startup_disabled_reason != null && 
+            <span style={{color:"#c50500"}} dangerouslySetInnerHTML={{__html: this.state.settings.general.startup_disabled_reason }} />
```

Comment:
There's nothing dangerous about this, since we supply the value `startup_disabled_reason` ourselves (and not from the internet).